account_statement_import_base/views/account_bank_statement_line.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
+<!--
+  Copyright 2022 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  Licence LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+-->
+<odoo>
+
+<record id="view_bank_statement_line_form" model="ir.ui.view">
+    <field name="model">account.bank.statement.line</field>
+    <field name="inherit_id" ref="account.view_bank_statement_line_form" />
+    <field name="arch" type="xml">
+        <field name="statement_id" position="attributes">
+            <attribute
+                    name="invisible"
+                >not context.get('statement_line_main_view')</attribute>
+        </field>
+        <xpath expr="//field[@name='company_id']/.." position="attributes">
+            <attribute name="col">2</attribute>
+        </xpath>
+        <field name="sequence" position="attributes">
```

Comment:
We meant all the opinionated hidings like this one and the following one. Please leave the standard as less touched as possible.

Reply:
I will take a screenshot before the PR and after the PR, and you will see that the form view is way better than before.